eventlogger/src/main/java/gov/cms/ab2d/eventlogger/LogManager.java

```diff
@@ -31,6 +40,57 @@ public void log(LoggableEvent event) {
         sqlEventLogger.updateAwsId(event.getAwsId(), event);
     }
 
+    /**
+     * Alert AB2D team via relevant loggers
```

Comment:
It looks like the new alert, trace, etc. are only for Slack and inserted into the main LogManager class without context for the other loggers.

Reply:
Yes, I don't see any other way to do it. Logs and alerts are different. There is a method specifying if you want to do both and a method specifying if you only want to do one or the other.